<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupPage</class>
 <widget class="QDialog" name="CreateGroupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="groupname_lbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Group Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupname_led">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="result_lbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>create Group result...</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancel_pbn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="creategroup_pbn">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
